src/Tests/Seam/When_receiving_messages.cs

```diff
@@ -1,95 +1,95 @@
 
-            var pump = new MessagePump(new TopologyOperator(receiversLifeCycleManager, converter, settings), receiversLifeCycleManager, converter, topologySectionManager, settings);
+//            var pump = new MessagePump(new TopologyOperator(receiversLifeCycleManager, converter, settings), receiversLifeCycleManager, converter, topologySectionManager, settings);
 
-            var completed = new AsyncManualResetEvent(false);
-            //var error = new AsyncManualResetEvent(false);
+//            var completed = new AsyncManualResetEvent(false);
+//            //var error = new AsyncManualResetEvent(false);
 
-            var received = false;
-            Exception ex = null;
+//            var received = false;
+//            Exception ex = null;
 
-            // Dummy CriticalError
-            var criticalError = new CriticalError(ctx => TaskEx.Completed);
+//            // Dummy CriticalError
+//            var criticalError = new CriticalError(ctx => TaskEx.Completed);
 
-            await pump.Init(context =>
-            {
-                received = true;
+//            await pump.Init(context =>
+//            {
+//                received = true;
 
-                completed.Set();
+//                completed.Set();
 
-                return TaskEx.Completed;
+//                return TaskEx.Completed;
 
-            }, null, criticalError, new PushSettings("sales", "error", false, TransportTransactionMode.ReceiveOnly));
+//            }, null, criticalError, new PushSettings("sales", "error", false, TransportTransactionMode.ReceiveOnly));
 
-            // execute
-            pump.Start(new PushRuntimeSettings(1));
+//            // execute
+//            pump.Start(new PushRuntimeSettings(1));
 
-            // send message to queue
-            var senderFactory = new MessageSenderCreator(messageFactoryLifeCycleManager, settings);
-            var sender = await senderFactory.Create("sales", null, "namespaceName");
-            await sender.Send(new BrokeredMessage());
+//            // send message to queue
+//            var senderFactory = new MessageSenderCreator(messageFactoryLifeCycleManager, settings);
+//            var sender = await senderFactory.Create("sales", null, "namespaceName");
+//            await sender.Send(new BrokeredMessage());
 
-            await completed.WaitAsync(cts.Token).IgnoreCancellation(); // Task.WhenAny(completed.WaitOne(), error.WaitOne());
+//            await completed.WaitAsync(cts.Token).IgnoreCancellation(); // Task.WhenAny(completed.WaitOne(), error.WaitOne());
 
-            // validate
-            Assert.IsTrue(received);
-            Assert.IsNull(ex);
+//            // validate
+//            Assert.IsTrue(received);
+//            Assert.IsNull(ex);
 
-            // cleanup
-            await pump.Stop();
-        }
-    }
-}
\ No newline at end of file
+//            // cleanup
+//            await pump.Stop();
+//        }
+//    }
+//}
```

Comment:
Why is this test commented out? Do you need to redo it? or does it need to be deleted?

Reply:
I need to make this compiling again but wanted to first hear your feedback before I proceed